_inc/client/state/connection/reducer.js

```diff
@@ -272,3 +272,13 @@ export function requiresConnection( state, slug ) {
 export function isUnavailableInDevMode( state, module ) {
 	return isDevMode( state ) && requiresConnection( state, module );
 }
+
+/**
+ * Checks if the JETPACK__SANDBOX_DOMAIN is set
+ *
+ * @param  {Object}      state Global state tree
+ * @return {bool|string} False if not sandboxed, string of URL if so.
```

Comment:
The default value for `JETPACK__SANDBOX_DOMAIN` is `""`. Will this function actually return `false` in that case, or will it return `""`?

Reply:
True! Updated to match reality.